scss/_base_code.scss

```diff
@@ -20,10 +21,10 @@
   }
 
   pre {
-    background: $color-x-dark;
-    border: 0;
-    border-radius: .5rem;
-    color: $color-mid-dark;
+    background-color: $color-light;
+    border: 1px solid $color-mid-dark;
+    border-radius: 2px;
```

Comment:
Not all units are relative, border is set to 1px. I think that if you zoom in a page where border radius are set in pixels the border radius is increased anyway. What else do you need to know?